filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
@plutoshe 

It seems that what you said is not what you have done (I probably misunderstood your code).

b is the data user provide.

1. You use a buffer to wrap around it to make it a reader.
2. You try to read at least chunk size.
3. If you reach an EOF during the read, you return `Need write content`.

Can you please tell me how this achieve what you said? 

If you want to make sure user give you something, why not simply check the length of `b` as I mentioned before?